cmd/monitor/monitor.go

```diff
@@ -104,7 +104,7 @@ func cmdFunc(cmd *cobra.Command, args []string) {
 	// start RPC server
 	server.Start()
 	// start Prometheus metrics server
-	metrics.Start()
+	metrics.Start(cfg.Metric.MetricServerPort)
```

Comment:
```suggestion
	metrics.Start(cfg.Metrics)
```

since in the future we may introduce more fields for the metrics config